### ISSUE TITLE
Error in ncumul_deceased in JU

### ISSUE THREAD

**Issue by rsalzer** [openZH/covid_19#1882]
<img width="1263" alt="image" src="https://user-images.githubusercontent.com/17248257/133224145-e8f6b15a-0897-4e3f-8024-0f534273ee44.png">

Only one day seems to be affected ... before and after the numbers are correct.

**Comment by andreasamsler** [comment]
JU is informed. Thanks.

**Comment by andreasamsler** [comment]
It is quite possible that JU's source on other days differs from the resource we scraped. We scrape what the cantons publish(ed). Establishing congruence retrospectively has its limits.

**Comment by rsalzer** [comment]
Well, congruence is quite easy, if the day before and the day after is the same ... it could just be corrected

**Comment by andreasamsler** [comment]
Nah, the complete time series would have to be made congruent with the official source of the canton; we asked JU, what their source would be.

**Comment by andreasamsler** [comment]
JU confirmed (via e-mail) an 'ncumul_deceased' of 151. Where do we scrape the values of that variable from - via https://www.jura.ch/fr/Autorites/Coronavirus/Infos-Actualite/Statistiques-COVID/Evolution-des-cas-COVID-19-dans-le-Jura.html or ?

**Comment by rsalzer** [comment]
Ok, thats two different issues:

This here is probably just a typo:
ncumul_deceased:
2021-02-07: 115 
2021-02-08: 8 
2021-02-09: 115

**Comment by maekke** [comment]
The number of deceased is summed up within the scraper from the table on the webpage. The dropping to 8 is a bug in the scraper, that I wasn't really able to spot/fix.
I've created a bit a different approach to the current JU scraper in #1846 that isn't perfect too, but should cover things. I've just updated it and the test run produces 145 deceased (also with summing it up), so the data isn't matching the expected number, but is at least a bit closer...
